src/driver/sqlx_mysql.rs

```diff
@@ -36,6 +36,7 @@ impl std::fmt::Debug for SqlxMySqlPoolConnection {
 
 impl SqlxMySqlConnector {
     /// Check if the URI provided corresponds to `mysql://` for a MySQL database
+    #[cfg(feature = "mock")]
```

Comment:
this is what confuses me, this `accepts` method should be in the public API, why does clippy think it's redundant?

Reply:
I think that is because the `accepts` method is not used from anywhere in this workspace except for `mock`. But if we want this method to be public, we need `#[allow(unused_variables)]` instead so I will replace them.

Reply:
Remove feature flag and I put `#[allow(unused_variables)]` instead
[09f845a](https://github.com/SeaQL/sea-orm/pull/2056/commits/09f845ad52fc6637ec25f66633ed6f04b6bb2153)